samples/samples-js/AddProductsNoPartialUpsert/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = async function (context) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+
+    var products = [];
+    for(let i = 0; i < 1000; i++) {
+        products.push(...{"productid": i,
```

Comment:
Is this working correctly? You're expanding the object here so I'm pretty sure this is going to end up with the values being inserted into the array instead of the item objects.